frontend/src/components/TableData.ts

```diff
@@ -1,9 +1,9 @@
 import { Component, Prop, Vue } from "vue-property-decorator";
 
 @Component({
-  name: "SQRTableData"
+  name: "TableData"
 })
-export default class SQRTableData extends Vue {
+export default class TableData extends Vue {
```

Comment:
Just curious: why `TableData` and not `DataTable`?

Reply:
DataTable.vue wraps v-data-table and TableData is used inside the `<td>` tag